validation/static/index.js

```diff
@@ -0,0 +1,143 @@
+"use strict";
+
+//    translation-judgement-accuracy-yes
+document.addEventListener('DOMContentLoaded', () => {
+    for (let judgement of ["yes", "no", "idk"]) {
+        for (let button of document.querySelectorAll(`.translation-judgement-accuracy-${judgement}`)) {
+            button.addEventListener("click", async (e) => {
+                const phraseId = e.target.dataset.phraseId
+
+                const response = await fetch(`/api/record_translation_judgement/${phraseId}`, {
+                    method: 'POST',
+                    mode: 'same-origin',    // Do not send CSRF token to another domain.
+                    headers: {
+                        'Content-Type': 'application/json',
+                        'X-CSRFToken': csrftoken
+                    },
+                    body: JSON.stringify({judgement})
+                })
+
+                let r = await response.json()
+
+                if (r.status != 'ok') {
```

Comment:
I recently learned that `fetch` responses have a simpler built-in property to check if ok or not:

```suggestion
                if (!r.ok) {
```

Reply:
This doesn't seem to work in this context 🤔

Reply:
That’s sad. I’ve seen it [in the docs](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) and want to try it but haven’t yet used it myself. 🤷

Reply:
Because`r` is the parsed HTTP body (JSON returned from the API); `response` is the HTTP response!

But that means `status` is part of our speech-db API! So I'm not sure if this is appropriate:

```suggestion
                if (response.status != 'ok') {
```